frontend/components/Editor.js

```diff
@@ -402,6 +406,9 @@ export class Editor extends Component {
         // these are update message that are _not_ a response to a `send(*, *, {create_promise: true})`
         const on_update = (update, by_me) => {
             if (this.state.notebook.notebook_id === update.notebook_id) {
+                if(this.state.save_medium) {
+                    this.state.save_medium.scheduleSave()
```

Comment:
I'm not sure if this is where this line should go under the new state management. This line schedules a notebook to be saved to gist. It is debounced, so it can be called many times without causing problems. However it should only be called by the client who made the changes. Otherwise saves will be scheduled on every open client corresponding to a session. Is that what `by_me` tracks? 

If so then `if(this.state.save_medium && by_me)` would fix this?